lib/fastlane/plugin/cordova/actions/cordova_action.rb

```diff
@@ -13,7 +13,8 @@ class CordovaAction < Action
         keystore_alias: 'alias',
         build_number: 'versionCode',
         min_sdk_version: 'gradleArg=-PcdvMinSdkVersion',
-        cordova_no_fetch: 'cordovaNoFetch'
+        cordova_no_fetch: 'cordovaNoFetch',
+        package_type: 'packageType'
```

Comment:
`type` makes more sense here, IMO. However, I figured it'd be easily conflated with the `type` param used for iOS (CORDOVA_IOS_PACKAGE_TYPE).